cluster/manifests/wiz/sensor-daemonset.yaml

```diff
@@ -0,0 +1,203 @@
+{{ if eq .Cluster.ConfigItems.wiz_enable_runtime_sensor "true"}}
+---
+# Source: wiz-sensor/templates/daemonset.yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: wiz-sensor
+  labels:
+    helm.sh/chart: wiz-sensor-1.0.4760
+    application: "wiz"
+    component: "sensor"
+  namespace: wiz
+spec:
+  selector:
+    matchLabels:
+      application: "wiz"
+      component: "sensor"
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        helm.sh/chart: wiz-sensor-1.0.4760
+        application: "wiz"
+        component: "sensor"
+      annotations:
+        container.apparmor.security.beta.kubernetes.io/wiz-sensor: unconfined
+        cluster-autoscaler.kubernetes.io/enable-ds-eviction: "true"
+        node-ready.cluster.zalando.org/exclude: "true"
+    spec:
+      serviceAccountName: wiz-sensor
+      nodeSelector:
+        {{ if eq .Cluster.ConfigItems.wiz_node_feature_rollout "true" }}
+        node-feature.zalando.org/wiz: enabled
+        {{ else }}
+        node.kubernetes.io/role: worker
+        {{ end }}
+      hostPID: true
+      hostIPC: true
+      tolerations:
+        - key: karpenter.sh/disruption
+          operator: Exists
+        - key: node.kubernetes.io/not-ready
+          operator: Exists
+        - key: aws.amazon.com/spot
+          operator: Exists
+        - key: zalando.org/node-not-ready
+          operator: Exists
+          effect: NoSchedule
+        - key: nvidia.com/gpu
+          value: present
+          effect: NoSchedule
+        - key: dedicated
+          value: skipper-ingress
+          effect: NoSchedule
+      restartPolicy: Always
+      containers:
+      - name: wiz-sensor
+        image: container-registry.zalando.net/secops-systems/wiz-sensor:v1-main-2
+        imagePullPolicy: IfNotPresent
+        securityContext:
+          capabilities:
+            add:
+            - SYS_ADMIN # moving between namespaces
+            - SYS_CHROOT # moving between namespaces
+            - SYS_RESOURCE # eBPF
+            - SYS_RAWIO # file hashing
+            - DAC_OVERRIDE # file hashing
+            - DAC_READ_SEARCH # file hashing
+            - NET_ADMIN # network events
+            - NET_RAW # network events
+            - IPC_LOCK # eBPF
+            - FOWNER # file hashing
+            - SYS_PTRACE # eBPF
+            - SYSLOG # kernel symbol resolve
```

Comment:
These capabilities should be enabled conditionally here: https://github.com/zalando-incubator/kubernetes-on-aws/blob/5f4fdd525d1fbabd0fbd717e387367c8843528ca/cluster/manifests/01-admission-control/config.yaml#L119-L136

(Those that are not already mentioned in the list)